### DISCUSSION TITLE
IOS Version 7.5.1 无pro授权入口,这个问题已经过去6个多月，请问还要再等多久呢？

### DISCUSSION THREAD

**Discussion by peisy** [getlantern/lantern-forum-cn#613] [General]
IOS Version 7.5.1 无pro授权入口,这个问题已经过去6个多月，请问还要再等多久呢？

**Comment by peisy** [comment]
@Mike777Khoo

**Reply by peisy** [reply]
@Mike777Khoo

**Reply by MaxMKMax** [reply]
@peisy 
您好. 有好消息了!!! 已经提交新版本给iOS商店发放了. 等通过. 应该这 3 天里面就能更新最新版本了.
非常抱歉让您久等了! 谢谢您的耐心等待和支持我们!

**Reply by peisy** [reply]
@Mike777Khoo Mike，又一周过去了，还是7.5.1版本

**Reply by MaxMKMax** [reply]
@peisy 已经提交新版本了. iOS迟迟不批准. 我们在想尽办法呢.

**Reply by peisy** [reply]
@Mike777Khoo Mike，又又一周过去了，还是7.5.1版本

**Comment by MaxMKMax** [comment]
![image](https://github.com/user-attachments/assets/b589aa82-920f-44c4-801c-4c27e1546c6f)

@peisy 您好. 您看, 5 月 15 日我就已经帮你们催了. 我们两个月前就提交给iOS了. 他们一直要我们改东改西的. 我已经在帮你们催技术团队了. 真的很抱歉让你们久等了. 我们也很无奈.

**Reply by peisy** [reply]
@Mike777Khoo  Hi Mike, 不管是什么原因，都已经过去了九个月，还是没有新版本，真的无法让我们理解
